src/containers/MapExplorer-SelectorPanel.jsx

```diff
@@ -133,7 +129,7 @@ class SelectorPanel extends Component {
           </div>
         </section>
         <section className="action-subpanel">
-          <button className="btn" onClick={this.updateMe}>(Apply)</button>
+          <button className="hidden" onClick={this.updateMe}>(Apply)</button>
```

Comment:
Slowly replacing "edit this one filter selection" option with a "edit ALL filter selections" feature